gfsphysics/GFS_layer/GFS_physics_driver.F90

```diff
@@ -660,6 +660,7 @@ subroutine GFS_physics_driver                                 &
       real    :: pshltr,QCQ,rh02
       real(kind=kind_phys), allocatable, dimension(:,:) :: den
 
+      real(kind=kind_phys) :: lndp_vgf  
```

Comment:
At some point we agreed to no longer accept changes to IPD unless they are backports of GFS v16 implementation bug fixes. Just mentioning it here, but I am not going to ask you to change anything.